packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -126,7 +117,7 @@ export interface EditableTree extends Iterable<EditableField> {
      * Sequences (including empty ones) are always exposed as {@link EditableField}s,
      * and everything else is either a single EditableTree or undefined depending on if it's empty.
      */
-    readonly [key: FieldKey]: UnwrappedEditableField;
+    [key: FieldKey]: UnwrappedEditableField;
```

Comment:
It would be good to update the doc comment to explain what the semantics of deleting or overwriting the field is with respect to concurrent edits: what happens if another client concurently inserts nodes in the field?

Reply:
@yann-achard-MS Updated for setting the value, added `TODO` to update docs regarding the delete semantics when you'll clarify this.

Reply:
For deletion of sequence fields, the semantics are "delete the set of existing nodes" so any concurrently inserted nodes will not be deleted.
For deletion of optional fields, the current semantics are also are "delete the set of existing nodes" but this will change to "clear the field (no matter what is in it)" once the optional field editor is used.